packages/devextreme/testing/tests/DevExpress.ui.widgets.pivotGrid/store.remote.tests.js

```diff
@@ -1565,7 +1565,7 @@ QUnit.module('Expanding items', moduleConfig, () => {
             };
             loadOptions[expandedPathOptionName] = [[2010], [2010, 1]];
             store.load(loadOptions).done(function() {
-                assert.deepEqual(actualFilter, [['date', '=', 2010]], 'rows count');
+                assert.deepEqual(actualFilter, ['date', '=', 2010], 'rows count');
             });
         });
```

Comment:
This test was to check if filter duplicates. Like `[['date', '=', '2010'], 'or', ['date', '=', '2010']]`. After my fix, the nesting structure is changed a little bit, but no duplicate filter occur, so that's ok